### ISSUE TITLE
the docker doesn't support 4090 GPU

### ISSUE THREAD

**Issue by whhong5** [microsoft/GLIP#144]
Great work! I tried to pull the docker image "pengchuanzhang/pytorch:ubuntu20.04_torch1.9-cuda11.3-nccl2.9.9" but when I run this image, I got the error message shown blow:

"WARNING: Detected NVIDIA NVIDIA GeForce RTX 4090 GPU, which is not yet supported in this version of the container"

I don't know how to solve this problem. Could you please give me some suggestions?

**Comment by ybk66** [comment]
可能是4090太先进了？

**Comment by liunian-harold-li** [comment]
Yeah, 4090 might be too advanced for our pre-built docker. You might need to manually install the dependencies.
